src/common/event_loop.cc

```diff
@@ -30,11 +30,13 @@ void event_loop_add_file(event_loop *loop,
   /* If it cannot be added, increase the size of the event loop. */
   if (err == AE_ERR && errno == ERANGE) {
     err = aeResizeSetSize(loop, 3 * aeGetSetSize(loop) / 2);
-    CHECK(err == AE_OK);
```

Comment:
That will fail, but it will also fail if the file descriptor is already closed, if I remember correctly. This can happen if file descriptor is closed by the peer while handling a message.